src/errorpage.cc

```diff
@@ -810,67 +811,59 @@ ErrorState::~ErrorState()
 int
 ErrorState::Dump(MemBuf * mb)
 {
-    MemBuf str;
-    char ntoabuf[MAX_IPSTRLEN];
+    PackableStream out(*mb);
+    SBufStream body;
+    const auto &encoding = CharacterSet::RFC3986_UNRESERVED();
+
+    body << "CacheHost: " << getMyHostname() << "\r\n" <<
```

Comment:
Do we have to change the _order_ of code snippets in this refactoring? The new body-then-header order is less natural. The order change complicates review. Please avoid this order change unless there is some good reason to change the existing order.

Reply:
Done. This version is in my opinion much more readable than the previous one
which intermingled appending to str and to mb